tests/customerSite/menu/viewCoffee/coffeeCupHasCorrectCost.spec.js

```
@@ -1,19 +1,35 @@
 import { test } from '../../../_fixtures/fixtures';
+import { allure } from 'allure-playwright';
```

This test also imports the module-level `allure` at the top. As with the other test file, this violates the requirement that metadata must be attached via the Playwright-bound per-test adapter: **"The labels must be attached in a way that is tied to the current test via the Playwright Allure adapter API so they appear in the report."**

Remove `import { allure } from 'allure-playwright'` and call the per-test adapter (`test.info().allure`) or expose the adapter via fixtures, and throw if missing so misconfiguration doesn't silently pass.